tutor/resources/styles/components/annotations/annotation.less

```diff
@@ -0,0 +1,113 @@
+tutor-highlight {
+  background-color: @tutor-highlight;
+}
+
+.annotater {
+  .widget {
+    background: @tutor-white;
+    border: thin solid @tutor-neutral-light;
+    font-size: 2rem;
+    padding: 1rem 0;
+    position: absolute;
+    transform: translate(calc(10px - 50%), 10px);
+    z-index: 20;
+    .no-select();
+
+    [role="button"] {
+      padding: 0 1.5rem;
+
+      &:first-child {
+        border-right: thin solid lightgray;
+      }
+    }
+  }
+
+  .arrow_box {
+
+    &:after,
+    &:before {
+      bottom: 100%;
+      left: 50%;
+      border: solid transparent;
+      content: " ";
+      height: 0;
+      width: 0;
+      position: absolute;
+      pointer-events: none;
+    }
+
+    &:after {
+      border-color: rgba(255, 255, 255, 0);
+      border-bottom-color: @tutor-white;
+      border-width: 12px;
+      margin-left: -12px;
+    }
+
+    &:before {
+      border-color: rgba(211, 211, 211, 0);
+      border-bottom-color: @tutor-neutral-light;
+      border-width: 13px;
+      margin-left: -13px;
+    }
+  }
+
+  .edit-box {
+    background-color: @tutor-neutral-bright;
+    border: thin solid @tutor-neutral-light;
+    padding: 1rem;
+    position: fixed;
+    right: 0;
+    top: 50%;
+    transition: 0.3s;
+    width: 40rem;
+    z-index: 20;
+    .tutor-shadow(subtle);
+
+    &.closed {
+      right: -40rem;
```

Comment:
Are these values calculated based on something else? Would it help to include a comment explaining why?

Reply:
They're chosen to try to emulate the UX mockup: http://mmwerz.axshare.com/#g=1&p=highlighting_-_no_tags